index.src.html

```diff
@@ -752,13 +757,30 @@ <h3 id="set-referrer-policy">
     </li>
   </ol>
 
+  <h3 id="set-requests-referrer-policy-on-redirect">
+    Set |request|'s referrer policy on redirect
+  </h3>
+
+  Given a {{Request}} |request| and a {{Response}} |actualResponse|,
```

Comment:
This `Request` should be [[concept-request]](https://fetch.spec.whatwg.org/#concept-request). Ditto for Response.